GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
That's fair. Although I don't know how to use GitVersion and the mere presence of the yaml file didn't seem to have any impact.
I'll wait to switch to GitVersion till the rest of the PR is reviewed and ready to merge.